components/ChatMessage.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Text, View, Image, StyleSheet } from 'react-native'
+
+const ChatMessage = props => {
+  return (
+    <View style={[props.message.from.email === "levans@email.com" ? styles.chat__user : styles.chat__friend]}>
```

Comment:
Applying class based on if the message if from the user itself or from another. Right now it's hardcoded, but once the login is running again, then we can fetch the user data from the store.